external/esp_idf_port/esp32/esp32_wifi_utils.c

```diff
@@ -56,6 +56,7 @@
 #include "nvs_flash.h"
 #include "esp_wifi_internal.h"
 
+#define WIFI_CONN_RETRY  5
```

Comment:
please make it indented to below defined macros

Reply:
tks, fixed